deeptools/plotHeatmap.py

```diff
@@ -186,7 +186,7 @@ def plotMatrix(hm, outFileName,
 
     if zMax is None:
         if matrix_flatten is None:
-            matrix_flatten = flattenMatrix(hm.matrix.matrix)
+            matrix_flatten = hm.matrix.matrix.flatten()
```

Comment:
I think the question was whether you instead meant `matrix_flatten = hm.matrix.flatten()`.